source/baikal/docker-install.md

```diff
@@ -1,85 +1,19 @@
 ---
-product: baikal 
-title: Dockerized Installation
+product: baikal
+title: Installation using Docker
 layout: default
 ---
 
-## Requirements
+Docker Image
+------------
```

Comment:
```suggestion
```

I'd take this additional header out completely - we almost have no text anyway and the new page title (Installation using Docker) already says it all.